rc/tools/git.kak

```diff
@@ -326,3 +334,59 @@ define-command -params 1.. \
             ;;
     esac
 }}
+
+# Options needed by git-diff-goto-source command
+declare-option str git_diff_hunk_filename
+declare-option int git_diff_hunk_line_num_start
+declare-option int git_diff_go_to_line_num
+declare-option str git_diff_git_dir
```

Comment:
You should make these options hidden, since they're just implementation details

Reply:
Done!